tests/unit/crsAdapter/TomaAdapter.test.js

```diff
@@ -224,6 +224,73 @@ describe('TomaAdapter', () => {
             });
         });
 
+        it('getData() should parse round-trip services', () => {
+            let xml = createTomaXml(
+                '<KindOfService.1>R</KindOfService.1>' +
+                '<Accommodation.1>YYZ</Accommodation.1>' +
+                '<ServiceCode.1>NEZE2784NQXTHEN</ServiceCode.1>' +
+                '<Name.1>DOE/JOHN</Name.1>' +
+                '<Reduction.1>040485</Reduction.1>' +
+                '<Title.1>H</Title.1>' +
+                '<From.1>051217</From.1>' +
+                '<Count.1>1</Count.1>' +
+                '<To.1>161217</To.1>'
+            );
+
+            let xml2 = createTomaXml(
+                '<KindOfService.1>R</KindOfService.1>' +
+                '<Accommodation.1>YYZ</Accommodation.1>' +
+                '<ServiceCode.1>NEZE2784NQXTHEN</ServiceCode.1>' +
+                '<Name.1>DOE/JOHN</Name.1>' +
+                '<Reduction.1>32</Reduction.1>' +
+                '<Title.1>H</Title.1>' +
+                '<From.1>051217</From.1>' +
+                '<Count.1>1</Count.1>' +
+                '<To.1>161217</To.1>'
+            );
+
+            let roundTripService1 = {
+                type: 'roundTrip',
+                bookingId: 'NEZE2784NQXTHEN',
+                destination: 'YYZ',
+                no: '1',
+                startDate: '05122017',
+                endDate: '16122017',
+                title: 'H',
+                name: 'DOE/JOHN',
+                birthdate: '040485'
+            };
+
+            let roundTripService2 = {
+                type: 'roundTrip',
+                bookingId: 'NEZE2784NQXTHEN',
+                destination: 'YYZ',
+                no: '1',
+                startDate: '05122017',
+                endDate: '16122017',
+                title: 'H',
+                name: 'DOE/JOHN',
+                age: '32'
+            };
+
+            TomaConnection.GetXmlData.and.returnValue(xml);
+
+            expect(adapter.getData()).toEqual({
+                services: [
+                    roundTripService1,
+                ]
+            });
+
+            TomaConnection.GetXmlData.and.returnValue(xml2);
+
+            expect(adapter.getData()).toEqual({
```

Comment:
if you test different use cases, then separate the tests and give them a name which infers the purpose of the test